deploy/kustomize/base/base.yaml

```diff
@@ -39,6 +39,20 @@ objects:
     # The bulk of your App. This is where your running apps will live
     deployments:
     -
+    jobs:
+    - name: koku-script-runner-job
+      podSpec:
+        name: koku-script-runner
+        image: ${IMAGE}:${IMAGE_TAG}
+        args:
+          - /bin/bash
+          - -c
+          - $HOME/scripts/run_aws_savings_plan_presto_migration.sh
```

Comment:
nitpick: let's make the name of the script something generic like `job_script.sh`. This way, if we need to run another job in the future, we just update that script and leave the clowdapp and cji alone.